general/networking/nginx.md

```diff
@@ -30,6 +30,11 @@ server {
     # listen [::]:443 ssl http2;
     server_name DOMAIN_NAME;
 
+    ## The following setting allows you to upload files like images.
+    sendfile on;
```

Comment:
I'd prefer to either remove this or explicitly set it to off since it's not a requirement to make uploads work and breaks uploading on filesystems like ZFS if enabled.

Reply:
Thank you for that input. I misunderstood how sendfile actually works.  📖👀
Since it is disabled (off) by default I would prefer to remove these two lines to keep it short and simple. 
If this is a common issue we might add an extra `## Troubleshooting` section for such things.

I do a quick rebase/force-push